src/test/regress/bin/normalize.sed

```diff
@@ -65,3 +68,17 @@ s/ERROR:  failed to execute task [0-9]+/ERROR:  failed to execute task X/g
 # normalize file names for partitioned files
 s/(task_[0-9]+\.)[0-9]+/\1xxxx/g
 s/(job_[0-9]+\/task_[0-9]+\/p_[0-9]+\.)[0-9]+/\1xxxx/g
+
+# Line info varies between versions
+/^LINE [0-9]+:.*$/d
+/^\s*\^\s*$/d
+
+# pg12 changes
+s/Partitioned table "/Table "/g
```

Comment:
Do you remember why we need `/Partitioned table "/Table "/g` and `invalid input syntax for type /invalid input syntax for /g`? 

If that's necessary for future changes, can we remove it for now and add whenever necessary?

(normilize.sed is super powerful and could potentially hide some bugs/issues etc, so I'm trying to be cautious while using it. )

I'd probably prefer to defer `Normalize for pg12` commit to future changes.

Reply:
These are changes in `\dt` & postgres error messages. It can make sense to separate the pg12 rules to the pg12 PR

Reply:
> These are changes in \dt & postgres error messages. It can make sense to separate the pg12 rules to the pg12 PR

Thanks, than let's remove those from this change before merging in.